admin/app/templates/views/platform-admin/callback_failure_summary.html

```diff
@@ -0,0 +1,35 @@
+{% extends "views/platform-admin/_base_template.html" %}
+{% from "components/page-footer.html" import page_footer %}
+{% from "components/previous-next-navigation.html" import previous_next_navigation %}
+{% from "components/table.html" import list_table, field, text_field, link_field %}
+
+{% block per_page_title %}
+  {{ page_title|capitalize }}
+{% endblock %}
+
+{% block platform_admin_content %}
+
+  <h1 class="heading-large">
+    {{ page_title }}
+  </h1>
+
+  {% call(item, row_number) list_table(
+    summary_rows,
+    caption="Services with failing callbacks",
+    caption_visible=False,
+    empty_message='Nothing to see',
+    field_headings=['Service', 'Fails'],
+    field_headings_visible=True
+  ) %}
+
+    {{ link_field(item[0], url_for('main.service_dashboard', service_id=item[1])) }}
```

Comment:
are we able to serialise out these records in a way that lets us have named fields to reference?

item[0] -> callback_failure.service_name
item[1] -> callback_failure.service_id
item[2] -> callback_failure.total_failure_count

Reply:
Yep, updated